src/commands/sbom/README.md

```diff
@@ -0,0 +1,47 @@
+# SBOM uploader
+
+This command lets you upload SBOM file to the Datadog intake endpoint.
+
+## Supported Formats
+
+ - CycloneDX 1.4
+
+## Usage
+
+```bash
+sbom  upload --service <my-service> <path/to/sbom.json>
+```
+
+### Environment variables
+
+The following environment variables must be defined
+
+ - `DD_SITE`: the [Datadog site](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site)
+ - `DD_APP_KEY`: the App Key to use
+ - `DD_API_KEY`: the API key to use
+ - `DD_SERVICE`: the Datadog service you use
```

Comment:
This doesn't need to be defined since you can pass in `--service some-service` though?